custom/opm/beneficiary.py

```diff
@@ -564,6 +564,24 @@ def weight_grade_normal(self):
                     return self.child_age/12
                 return False
 
+    def weight_grade_status(self, status):
+        if self.status == 'pregnant':
+            return False
+        if self.child_index == 1:
+            form_prop = 'interpret_grade'
+        else:
+            form_prop = 'interpret_grade_{}'.format(self.child_index)
+
+        forms = self.filtered_forms(CHILDREN_FORMS, 1)
+        if len(forms) == 0:
+            return False
```

Comment:
That's a good point.  We don't know enough to say whether or not the child is normal weight.  Think it'll be confusing if SAM + MAM + Normal != total_children?  Should we reduce the denominator for these columns to only include children with forms?